pom.xml

```diff
@@ -367,54 +367,58 @@
+                                <!-- Add files where to include licensing -->
+                                <include>src/**/*.java</include>
+                                <include>src/**/*.xml</include>
+                                <include>src/**/*.proto</include>
+                                <include>**/*.css</include>
+                                <include>**/*.sh</include>
+                            </includes>
+                            <excludes>
+                                <!-- Add files to exclude from licensing -->
 
-                        <!-- Exclude pom -->
-                        <exclude>**/pom.xml</exclude>
+                                <!-- Exclude pom -->
+                                <exclude>**/pom.xml</exclude>
 
-                        <!-- Exclude config and scripts -->
-                        <exclude>**/dev-utils/*</exclude>
-                        <exclude>dev-support/artifacts/**</exclude>
-                        <exclude>dev-support/work/**</exclude>
+                                <!-- Exclude config and scripts -->
+                                <exclude>**/dev-utils/*</exclude>
+                                <exclude>dev-support/artifacts/**</exclude>
+                                <exclude>dev-support/work/**</exclude>
 
 
-                        <!-- Exclude assembly -->
-                        <exclude>**/maven/assembly/*</exclude>
+                                <!-- Exclude assembly -->
+                                <exclude>**/maven/assembly/*</exclude>
 
-                        <!-- Project specific exclusions (e.g. they've been copied/modified from other open-source projects -->
+                                <!-- Project specific exclusions (e.g. they've been copied/modified from other open-source projects -->
 
-                        <!-- Protocol buffers generated classes -->
-                        <exclude>**/src/main/java/org/apache/omid/proto/TSOProto.java</exclude>
+                                <!-- Protocol buffers generated classes -->
+                                <exclude>**/src/main/java/org/apache/omid/proto/TSOProto.java</exclude>
 
-                        <!-- Taken from https://github.com/brianfrankcooper/YCSB -->
-                        <exclude>**/src/main/java/org/apache/omid/benchmarks/utils/Generator.java</exclude>
-                        <exclude>**/src/main/java/org/apache/omid/benchmarks/utils/IntegerGenerator.java</exclude>
-                        <exclude>**/src/main/java/org/apache/omid/benchmarks/utils/ZipfianGenerator.java</exclude>
-                        <exclude>**/src/main/java/org/apache/omid/benchmarks/utils/ScrambledZipfianGenerator.java
-                        </exclude>
+                                <!-- Taken from https://github.com/brianfrankcooper/YCSB -->
+                                <exclude>**/src/main/java/org/apache/omid/benchmarks/utils/Generator.java</exclude>
```

Comment:
These should not be excluded.
Instead, the if the license is indeed ASL compatible (I think it is), then it should be added as a valid license.